src/methods/bbknn_ts/config.vsh.yaml

```
@@ -0,0 +1,54 @@
+__merge__: ../../api/comp_method.yaml
+
+name: bbknn_ts
+label: BBKNN (TS)
+summary: "A combination of ComBat and BBKNN discovered and implemented by Gemini."
+description: |
+  "The BBKNN (TS) solution applies standard scRNA-seq preprocessing steps, including total count normalization, log-transformation, and scaling of gene expression data. Batch effect correction is performed using scanpy.pp.combat directly on the gene expression matrix (before dimensionality reduction). Dimensionality reduction is then applied using PCA on the ComBat-corrected data, and this PCA embedding (adata.obsm['X_pca']) is designated as the integrated embedding (adata.obsm['X_emb']). A custom batch-aware nearest neighbors graph is constructed based on this integrated embedding; for each cell, neighbors are independently identified within its own batch and other batches, up to n_neighbors_per_batch. These candidate neighbors are merged, keeping the minimum distance for duplicate entries, and the top total_k_neighbors are selected for each cell. Finally, a symmetric sparse distance matrix and a binary connectivities matrix are generated to represent the integrated neighborhood graph. This code was entirely written by the AI system described in the associated publication."
+references:
+  bibtex: |
+    @misc{GoogleScienceAI2025,
+      title={An {AI} system to help scientists write expert-level empirical software},
+      author={Eser Ayg\"un and Anastasiya Belyaeva and Gheorghe Comanici and Marc Coram and Hao Cui and Jake Garrison and Renee Johnston and Anton Kast and Cory Y. McLean and Peter Norgaard and Zahra Shamsi and David Smalling and James Thompson and Subhashini Venugopalan and Brian P. Williams and Chujun He and Sarah Martinson and Martyna Plomecka and Lai Wei and Yuchen Zhou and Qian-Ze Zhu and Matthew Abraham and Erica Brand and Anna Bulanova and Jeffrey A. Cardille and Chris Co and Scott Ellsworth and Grace Joseph and Malcolm Kane and Ryan Krueger and Johan Kartiwa and Dan Liebling and Jan-Matthis Lueckmann and Paul Raccuglia and Xuefei (Julie) Wang and Katherine Chou and James Manyika and Yossi Matias and John C. Platt and Lizzie Dorfman and Shibl Mourad and Michael P. Brenner},
+      year={2025},
+      eprint={2509.06503},
+      archivePrefix={arXiv},
+      primaryClass={cs.AI},
+      url={https://arxiv.org/abs/2509.06503}
+    }
+links:
+  documentation: https://google-research.github.io/score/
+  repository: https://github.com/google-research/score
+
+info:
+  method_types: [embedding]  
```

Since BBKNN only modifies the KNN graph only, the ouput type should be `knn`, not `embedding`
```suggestion
  method_types: [knn]  
```

If you also want to consider the Combat-corrected counts as additional part of the method output and configure, both outputs as folllows:
```suggestion
  method_types: [full, knn]  
```
the evaluation pipeline considers it as a separate version of the method and evaluate it separately.
So there will be an evaluation of `bbknn_ts:full` and `bbknn_ts:knn`, but there won't be any combination of metrics from the Combat corrected counts and BBKNN in a single entry in the results table.

Either way, embedding isn't the correct output format of this approach, because the resulting KNN graph from the embedding will be used for knn-based metrics, not the BBKNN corrected graph